core/consensus/sumeragi.hpp

```diff
@@ -35,14 +35,17 @@ namespace sumeragi {
     );
     void loop();
 
+    template<typename T,typename U>
     void getNextOrder(
-            std::unique_ptr<consensus_event::ConsensusEvent> const event
+            std::unique_ptr<consensus_event::ConsensusEvent<T,U>> const event
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Function parameter 'event' should be passed by reference.](https://www.codacy.com/app/soramitsu/iroha/file/4345715860/issues/source?bid=3712491&fileBranchId=3870389#l40)